docker-compose.yml

```
@@ -41,5 +41,29 @@ services:
+  azurite:
+    image: mcr.microsoft.com/azure-storage/azurite
+    command: azurite --blobHost 0.0.0.0 --skipApiVersionCheck
+    ports:
+      - "10000:10000"
+    volumes:
+      - azurite-data:/data
+    extra_hosts:
+      - "localhost:host-gateway"
+    networks:
+      - portabase
+    #  DefaultEndpointsProtocol=http;
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)